cmd/org/common.go

```diff
@@ -0,0 +1,62 @@
+package org
+
+import (
+	"fmt"
+	"os"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/openshift/osdctl/pkg/printer"
+	"github.com/spf13/cobra"
+)
+
+const (
+	organizationsAPIPath  = "/api/accounts_mgmt/v1/organizations/"
+	accountsAPIPath       = "/api/accounts_mgmt/v1/accounts"
+	currentAccountApiPath = "/api/accounts_mgmt/v1/current_account"
+)
+
+type Organization struct {
+	ID           string `json:"id"`
+	ExternalID   string `json:"external_id"`
+	Name         string `json:"name"`
+	EBSAccoundID string `json:"ebs_account_id"`
+	Created      string `json:"created_at"`
+	Updated      string `json:"updated_at"`
+}
+
+func sendRequest(request *sdk.Request) (*sdk.Response, error) {
+	response, err := request.Send()
+	if err != nil {
+		return nil, fmt.Errorf("cannot send request: %q", err)
+	}
+	return response, nil
+}
+
+func checkOrgId(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
```
We wouldn't want to print the help message for such scenarios.

Reply:
done